app/src/main/java/com/topjohnwu/magisk/core/tasks/MagiskInstaller.kt

```diff
@@ -412,10 +443,9 @@ abstract class MagiskInstallImpl protected constructor(
 
     protected fun doPatchFile(patchFile: Uri) = extractFiles() && handleFile(patchFile)
 
-    protected fun direct() = findImage() && extractFiles() && patchBoot() && flashBoot()
+    protected fun direct() = findImage() && extractFiles() && addond() && patchBoot() && flashBoot()
 
-    protected suspend fun secondSlot() =
```

Comment:
Could you revert the formatting change here please?
Everything that changes can cause merging conflicts later in CI.

Reply:
Done